src/app/routes/auth.ts

```diff
@@ -67,7 +67,9 @@ export class AuthController {
   async access(req: Request, res: Response) {
     const MAX_LOGIN_FAIL_ATTEMPTS = 10;
 
-    const userData: any = await this.service.User.FindUserByEmail(req.body.email);
+    const userData: any = await this.service.User.FindUserByEmail(req.body.email).catch(() => {
+      // eslint-disable @typescript-eslint/no-empty-function
```

Comment:
i'm not sure I understand this 🤔 if the promise fails we want it to fail silently?

Reply:
yeah, provably should added the code the follows that line 😅. 
The aim of this change is to let the controller handle the case to return an `401 Unauthorized`
https://github.com/internxt/drive-server/blob/ea5e8fae0c96497852a6491fcb772cbeb1eaf02e/src/app/routes/auth.ts#L70-L74

I could moved that logic to the catch though

Reply:
Oh I see what you mean, nice! 

Perhaps would be good to log an attempt to login with a non-existing email, can you please add something like:
`logger.log('Attempted login with a non-existing email: %s', req.body.email)`